src/reformat/rgb.rs

```diff
@@ -457,4 +543,28 @@ mod tests {
         }
         Ok(())
     }
+
+    #[test]
+    fn shuffle_channels_to() {
+        let image = Image {
+            width: 1,
+            height: 1,
+            depth: 8,
+            format: Format::Rgba,
+            pixels: Some(Pixels::Buffer(vec![0u8, 1, 2, 3])),
+            row_bytes: 4,
+            ..Default::default()
+        };
+
+        for (format, expected) in [
```

Comment:
please make this a test parameter instead of a for loop. so that we get all the benefits (each test will run in parallel, know exactly which one failed, etc).

something like this:

```rust
use test_case::test_case;
#[test_case(Format::Rgba, &[0, 1, 2, 3])]
#[test_case(Format::Abgr, &[3, 2, 1, 0])]
#[test_case(Format::Rgb, &[0, 1, 2])]
fn shuffle_channels_to(format: Format, expected: &[u8]) {
  let image = ...;
  assert_eq!(...); // use format and expected here.
}
```

untested code. but there should be other examples in the repo that uses the test_case for generating parameterized tests.